ethereum.star

```diff
@@ -1,13 +1,13 @@
 ethereum_package = import_module(
-    "github.com/ethpandaops/ethereum-package/main.star@1ad949f4f65a34f041bd90050ca407e370eee579"  # 2025-04-11
+    "github.com/ethpandaops/ethereum-package/main.star@197cdf84cbcc713f46bb37c4af84c4a0cf1854ff"  # 2025-05-28
 )
 
-GETH_IMAGE = "ethereum/client-go:v1.14.12"
+GETH_IMAGE = "ethereum/client-go:v1.15.11"
 # There's an issue with the latest version of the ethereum-package and lighthouse minimal image.
 # https://github.com/ethpandaops/ethereum-package/issues/899
 # The fix is not ideal for now since we're waiting on lighthouse to push a fix image.
 # https://github.com/ethpandaops/ethereum-package/pull/915
-LIGHTHOUSE_IMAGE = "ethpandaops/lighthouse:unstable"
+LIGHTHOUSE_IMAGE = "ethpandaops/lighthouse:stable"
```

Comment:
Just to make sure it doesn't break in the future:

![Screenshot 2025-06-02 at 09 55 15](https://github.com/user-attachments/assets/658c6cea-b866-4947-af54-e304610ed184)


```suggestion
LIGHTHOUSE_IMAGE = "ethpandaops/lighthouse:stable-999b045"
```